lib/bolt/cli/src/commands/admin/cluster/mod.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::*;
+use bolt_core::{
+	context::ProjectContext,
+	tasks::{self, api::send_request},
+	utils,
+};
+use clap::Parser;
+use rivet_api::{
+	apis::{configuration::Configuration, *},
+	models,
+};
+use uuid::Uuid;
+
+mod datacenter;
+
+#[derive(Parser)]
+pub enum SubCommand {
+	/// Creates a new cluster
+	Create {
+		/// The name of the cluster
+		#[clap(short, long)]
+		name: String,
+		/// The ID of the owner team
+		#[clap(short, long)]
+		owner_team_id: String,
+	},
+	/// Deletes an existing cluster
+	Delete {
+		/// The ID of the cluster to delete
+		#[clap(short, long)]
+		cluster_id: String,
+	},
+	/// Creates a new datacenter
+	Datacenter {
+		#[clap(subcommand)]
+		command: datacenter::SubCommand,
+	},
+}
+
+impl SubCommand {
+	pub async fn execute(self, ctx: ProjectContext, config: Configuration) -> Result<()> {
+		match self {
+			Self::Create {
+				name,
+				owner_team_id,
+			} => {
+				admin_cluster_api::admin_cluster_create(
```

Comment:
should verify that provisioning is enabled in the ns config

Reply:
Is `ctx.ns().rivet.provisioning.is_some()` a valid way to check this?

Reply:
Yes, though i would use `ensure!` to throw an error